workflows/funcscan.nf

```diff
@@ -17,6 +17,9 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 // Check mandatory parameters
 if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
 
+// TODO update to only display if DeepARG is activated
+if (  "${workflow.containerEngine}" == 'singularity' && params.deeparg_data ) log.warn("[nf-core/funcscan] warning: running with singularity requires DeepARG to run the container with --fakerun. This may not be available on all systems")
```

Comment:
No need for string interpolation and run option.

```suggestion
if (  workflow.containerEngine == 'singularity' && params.deeparg_data && params.run_deeparg ) log.warn("[nf-core/funcscan] warning: running with singularity requires DeepARG to run the container with --fakerun. This may not be available on all systems")
```